antarest/study/storage/variantstudy/variant_study_service.py

```diff
@@ -884,6 +884,7 @@ def _generate(
             study = self.study_factory.create_from_fs(
                 self.get_study_path(variant_study),
                 study_id=variant_study.id,
+                output_path=Path(variant_study.path) / OUTPUT_RELATIVE_PATH,
```

Comment:
Sometimes we use `OUTPUT_RELATIVE_PATH`, and sometimes we use `"outpu"`.
What's the difference ? Why not always use `OUTPUT_RELATIVE_PATH` ?

Reply:
I don't know where we use "output" (not related to this PR), but yes we should use `OUTPUT_RELATIVE_PATH` instead of `"output"` in most cases I guess